class RequestFault(Exception):
    """Исключение при ошибке запроса."""

    pass
